test/e2e/scheduling/priorities.go

```diff
@@ -255,6 +255,17 @@ var _ = SIGDescribe("SchedulerPriorities", framework.WithSerial(), func() {
 		tolePod, err := cs.CoreV1().Pods(ns).Get(ctx, tolerationPodName, metav1.GetOptions{})
 		framework.ExpectNoError(err)
 		gomega.Expect(tolePod.Spec.NodeName).To(gomega.Equal(nodeName))
+
+		ginkgo.By("Pod should contain the PodScheduled condition and its manager should be kube-scheduler")
+		managedFields := tolePod.GetManagedFields()
+		PodScheduledConditionFound := false
+		for _, entry := range managedFields {
+			if entry.FieldsV1.String() == `{"f:status":{"f:conditions":{".":{},"k:{\"type\":\"PodScheduled\"}":{".":{},"f:lastProbeTime":{},"f:lastTransitionTime":{},"f:status":{},"f:type":{}}}}}` {
```

Comment:
This may lead to subsequent minor changes causing the tests to fail.

I'm thinking maybe a simple `if strings.Contains(entry.FieldsV1.String(),"PodScheduled")` is enough?